### PR TITLE
Changed helm repos, added postgresql for IP

### PR DESCRIPTION
**Pull request by roberthstrand**

### CODE REVIEW

File: workloads/cert-manager/cert-vault.innovasjonnorge.no.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: cert-manager.io/v1alpha3
 kind: Certificate
 metadata:
   name: vault-innovasjonnorge-no
-  namespace: vault
```
Comment (Happycoil) [inline workloads/cert-manager/cert-vault.innovasjonnorge.no.yml:5]: Unrelated change?

Comment (roberthstrand) [reply workloads/cert-manager/cert-vault.innovasjonnorge.no.yml:5]: Changed manually when changing namespace for vault, this code reflects that. Shouldn't be an issue ;)

**Review by Happycoil — APPROVED** [review]
:shipit:
